src/styles/index.scss

```
@@ -1,3 +1,125 @@
+@import './variables';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  display: flex;
+}
+
+.card {
+  width: 200px;
+  height: 406px;
+  padding: 16px;
+  border-radius: 5px;
+  color: $main-text-color;
+  box-sizing: border-box;
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+}
+
+.describe {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  margin-top: 40px;
+}
+
+.code {
+  font-size: 10px;
+  line-height: 14px;
+  color: $secondary-color;
+  margin: 0;
+  margin-bottom: 16px;
+}
+
+.rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  text-align: center;
+
+  .reviews {
+    color: $main-text-color;
+    font-size: 10px;
+    line-height: 14px;
+    width: 400px;
+    margin: 0;
+    text-align: right;
+  }
+}
+
+.stars {
+  display: flex;
+  width: 100%;
+  max-width: 250px;
+  justify-content: flex-start;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-size: cover;
+    margin-right: 4px;
+  }
+
+  &--4 .stars__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+.price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  line-height: 18px;
+  height: 18px;
+  width: 100%;
+  margin-bottom: 16px;
+  margin-top: 24px;
+
+  &-text {
+    color: $secondary-color;
+    font-size: 12px;
+    margin: 0;
+  }
+
+  h2 {
+    font-weight: 700;
+    font-size: 16px;
+    margin: 0;
+  }
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid $button-color;
+  border-radius: 5px;
+  width: 164px;
```

do not hardcode width in this case
```suggestion
  width: 100%;
```